pybamm/input/parameters/lithium_ion/negative_electrodes/graphite_Chen2020/graphite_LGM50_ocp_Chen2020.py

```diff
@@ -26,10 +27,11 @@ def graphite_LGM50_ocp_Chen2020(sto):
 
     u_eq = (
         1.9793 * exp(-39.3631 * sto)
-        + 0.2482
+        + 0.15561
```

Comment:
We need to be careful that this is still representative of the results in the paper, tagging @brosaplanella